ush/eva/marine_eva_post.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+import argparse
+import datetime
+import logging
+import os
+import socket
+import yaml
+import glob
+
+
+def marine_eva_post(inputdir, outputdir, diagdir):
+    logging.basicConfig(format='%(asctime)s:%(levelname)s:%(message)s', level=logging.INFO, datefmt='%Y-%m-%d %H:%M:%S')
+    inputyamls = glob.glob(os.path.join(inputdir, '*.yaml'))
```

Comment:
Let's make that function operate on 1 yaml only, while diagdir is going to be common to all, that isn't true for the bounds for example. Simpler to call this function with specific argument for a specific obs type.

Reply:
Ok, sounds safer